### PR TITLE
consolidate builds

### PR DESCRIPTION
**Pull request by jdanyow**
no need for separate builds *yet* if they only differ by the last step

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -21,11 +25,7 @@ jobs:
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-      - uses: actions/checkout@v2
-
-      # Runs a set of commands using the runners shell
-      - name: Go to workspace
```
Comment (jdanyow) [inline .github/workflows/main.yml:27]: ought to already be in the workspace dir

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -47,4 +47,5 @@ jobs:
 
       # If the version has been incremented (with `yarn changeset version`), publish package
       - name: Publish on version increment
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
```
Comment (jdanyow) [inline .github/workflows/main.yml:50]: the ref check for main is to future proof this line if we expand the pr branch matching  on line 11/12/13

Comment (wibjorn) [reply .github/workflows/main.yml:50]: I think I'd prefer to keep the build separate

Comment (wibjorn) [reply .github/workflows/main.yml:50]: I think I'd prefer to keep the build separate

**Comment by wibjorn** [comment]
Works! https://github.com/microsoft/atlas-design/runs/2117039762?check_suite_focus=true

**Review by wibjorn — APPROVED** [review]

**Pull request merged**